eth_p2p.nim

```diff
@@ -58,10 +58,16 @@ proc processIncoming(server: StreamServer,
   yield peerfut
   if not peerfut.failed:
     let peer = peerfut.read()
-    echo "TODO: Add peer to the pool..."
+    if node.peerPool != nil:
+      if peer.remote notin node.peerPool.connectedNodes:
```

Comment:
This snipped below can be extracted in a `PeerPool.addPeer` or `PeerPool.peerAdded` proc. I see roughly the same code in `PeerPool.connectToNode`.